Automattic/newspack-plugin
# Use CSS grid to implement the Muriel grid

### All Submissions:






### Changes proposed in this Pull Request:



Introduces an approach to address #171 using CSS grid to size and position components to the Muriel grid.

Before:
![image](https://user-images.githubusercontent.com/5375500/60857894-44c1ba00-a1da-11e9-9552-94f99ec4e534.png)

After: 
![image](https://user-images.githubusercontent.com/5375500/60857886-3ecbd900-a1da-11e9-8fbe-4ef4516a1535.png)

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Navigate to `Newspack -> Components demo`
3. Verify that the components align to the grid at all viewport sizes

Note that these changes attempt to scope this to the `Components demo` page only. Thus having to add the `muriel-grid-item` classname to each card component in the components demo.

### Other information:
